artemis-server/src/test/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImplTest.java

```diff
@@ -1082,6 +1083,68 @@ public void testAddressSettingsPageLimitInvalidConfiguration2() throws Throwable
 
       String randomString = RandomUtil.randomString();
 
+      properties.put("addressSettings.#.expiryAddress", randomString);
+      properties.put("addressSettings.#.pageLimitMessages", "300");
+      // properties.put("addressSettings.#.pageLimitBytes", "300000"); out on purpose
+      //properties.put("addressSettings.#.pageFullMessagePolicy", "DROP"); // removing the pageFull on purpose
```

Comment:
The commented out code seems out of place (and inconsistently indented)...why not just have the comment explain pageLimitBytes and pageFullMessagePolicy are missing to test the validation of them being missing. Which the existing comments dont even really make clear.

Reply:
to me, the commented out code makes it clear on the intention.. but fine, I will remove it.